packages/forma-36-react-components/src/components/DateTime/DateTime.test.tsx

```diff
@@ -51,9 +51,12 @@ describe('DateTime', () => {
   it('does not allow for an unknown format', () => {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const format = 'NOT_REAL' as any;
-
+    // Throw error silently
+    const spy = jest.spyOn(console, 'error');
+    spy.mockImplementation(() => {});
     expect(() => {
       render(<DateTime date={exampleDate} format={format} />);
     }).toThrow(`Unknown date format 'NOT_REAL'`);
+    spy.mockRestore();
```

Comment:
I'd love to silence this error, but is it jest's default behaviour to log an error that's expected, or is it our code that does it? If it's the former I'm not sure we should mess with it 🤔

Reply:
[Here](https://github.com/facebook/jest/issues/5785) is a Github issue thread on this topic. The OP claims that he's seeing the expected errors in the console during development which makes it hard to find out if the tests actually passed. Looking at the following discussion it appears not to be expected behaviour. One guess is related to async/ sync tests.
My solution is not perfect since it solves this problem only for this specific case and if the test fails we might not see the error. Still, I believe that this kind of logging is not useful and should be omitted until required for some reason. This makes reading test results way easier.

Reply:
Thanks for the context! Silencing the error seems the way to go, but can we maybe add a type check here to make sure we only silence that specific error?

Reply:
That's a good idea. Then, we will see again if something unexpected happens :)

Reply:
I created a util function `monkeyPatch()` so we can maybe reuse that later at another point.